data/links.csv

```diff
@@ -148,13 +148,13 @@ dateAdded,url,title
 2021-09-17,https://securityboulevard.com/2021/09/some-recent-api-security-related-gaffes-and-how-they-might-have-been-avoided/,"Some Recent API Security Related Gaffes, And How They Might Have Been Avoided - Security Boulevard"
 2021-09-17,https://www.infoq.com/news/2021/09/api-design-first-addr/,"API Design-First Using the ""Align-Define-Design-Refine"" Process"
 2021-09-24,https://apievangelist.com/2021/09/20/building-an-api-platform-that-will-support-the-future/,Building an API Platform That Will Support the Future
-2021-09-24,https://blog.dream11engineering.com/lessons-learned-from-running-graphql-at-scale-2ad60b3cefeb,"Lessons learned from running GraphQL at scale | Dream11 Engineering"
+2021-09-24,https://blog.dream11engineering.com/lessons-learned-from-running-graphql-at-scale-2ad60b3cefeb,Lessons learned from running GraphQL at scale | Dream11 Engineering
```

Comment:
Not sure these changes have meant to come in?

Reply:
Ugh, there's always someone slapdash. Hang on, I'll have a look.